mods/ascent/mobs/bodyparts_insectoid.dm

```diff
@@ -77,4 +77,31 @@
 
 /obj/item/organ/internal/eyes/insectoid
 	name = "compound ocelli"
-	icon_state = "eyes-compound"
\ No newline at end of file
+	icon_state = "eyes-compound"
+
+/obj/item/organ/internal/egg_sac/insectoid
+	name = "gyne egg-sac"
+	action_button_name = "Produce Egg"
+	var/egg_metabolic_cost = 100
+
+/obj/item/organ/internal/egg_sac/insectoid/refresh_action_button()
+	. = ..()
+	if(.)
+		action.button_icon_state = "serpentid-threat"
```

Comment:
Deliberate?

Reply:
Yeah, trying to figure out how to use action button icons on /item. It's a bit odd.